routes/ffxiv.py

```
@@ -46,3 +46,54 @@
+    # DEPRECIATED
+    if request.method == "GET":
+        return return_safe_html(render_template("petshoppinglist.html"))
+    elif request.method == "POST":
+        json_data = {
+            "region": request.form.get("region"),
+            "itemID": int(request.form.get("petID")),
+            "maxPurchasePrice": int(request.form.get("maxPurchasePrice")),
+            "connectedRealmIDs": {},
+        }
+
+        response = requests.post(
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)